backend/packages/Upgrade/src/api/services/ExperimentService.ts

```diff
@@ -1619,8 +1619,11 @@ export class ExperimentService {
     return errorString;
   }
 
-  private deduceExperimentDetails(experiment: ExperimentDTO): ExperimentDTO {
+  private deduceExperimentDetails(experiment: Experiment): Experiment {
     experiment.id = uuid();
+    // set current createdAt date and initialize versionNumber as fresh experiment version to detect updates
+    experiment.createdAt = new Date();
```

Comment:
Instead of setting the value, we should delete the value and let TypeORM set the value.